### PR TITLE
Update sb-deprecation.md

### PR DESCRIPTION
**Pull request by carsonruebel**
Updating date to 11/1/2021 and changed verbiage to reflect future tense rather than past tense.

### CODE REVIEW

**Comment by carsonruebel** [comment]
Closing and reopening for unrelated validation warning per https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution%2Cpr-merge-conflict-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch

**Review by mikefrobbins — APPROVED** [review]
LGTM

**Comment by mikefrobbins** [comment]
Following up with @dcaro prior to merging this PR.

**Comment by carsonruebel** [comment]
Checking back on this PR. Is this good to merge?

File: azuresmps-4.0.0/include/sb-deprecation.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ ms.date: 05/25/2020
 ms.topic: include
 ---
 > [!IMPORTANT]
-> This PowerShell command for Service Bus is no longer supported as of **11/1/2019**. The Azure
-> Service Management model is deprecated for Service Bus and was disabled on that date. Please
+> This PowerShell command for Service Bus is no longer supported as of **11/1/2021**. The Azure
```
Comment (dcaro) [inline azuresmps-4.0.0/include/sb-deprecation.md:7]: The Service Management REST API for Service Bus will be retired by November 1, 2021. Go to the [deprecation of Azure Service Manager](https://docs.microsoft.com/en-us/azure/service-bus-messaging/deprecate-service-bus-management) for more information about the retirement of the API.
This Azure Service Management PowerShell cmdlet for Service Bus will no longer be supported as of November 1, 2021. Please migrate to the new Azure PowerShell cmdlets (https://docs.microsoft.com/powershell/module/az.servicebus/)

**Review by dcaro — CHANGES_REQUESTED** [review]
@carsonruebel per this [article](https://techcommunity.microsoft.com/t5/messaging-on-azure/deprecating-service-management-support-for-azure-service-bus/ba-p/370909) and post on [Azure update] (https://azure.microsoft.com/en-us/updates/service-management-rest-apis-and-support-will-be-retired-on-january-11-2021/) my understanding is that the RDFE cmdlets for service bus will retire by 10/31/2021.
Can you please update the include with the following:
- Add links to the documentation associated to the retirement,
- Have a pointer to the new cmdlets
- Since this has not happened yet, using the the past is akward.

File: azuresmps-4.0.0/include/sb-deprecation.md (hunk 1 above)
Comment (carsonruebel) [reply azuresmps-4.0.0/include/sb-deprecation.md]: Updated the text per suggestion

**Pull request merged**